src/components/SettingsModal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import reactDom from 'react-dom';
+import './SettingsModal.scss';
+
+const SettingsModal = ({ isOpen, onClose }) => {
+  if (!isOpen) {
+    return null;
+  }
+
+  return reactDom.createPortal(
+    <div className={'SettingsModal__overlay'}>
```

Comment:
If there's no variable involved you should always just use pure string syntax instead. Same as all the other className in this project.
```suggestion
    <div className="SettingsModal__overlay">
```